react-components/src/data-providers/core-dm-provider/check3dConnectedEquipmentQuery.ts

```diff
@@ -193,3 +197,48 @@ export function createCheck3dConnectedEquipmentQuery(
     }
   } as const satisfies Omit<QueryRequest, 'parameters' | 'cursor'>;
 }
+
+// We want to preserve most of the typing of the object returned here,
+// as it gives us better types in the result of the query call later on
+// Specifying a type here would either mean losing valuable type information
+// or an enourmous type
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+export function createCheck3dConnectedEquipmentQueryNoIndirect(
```

Comment:
non blocking
I would suggest to rename it to `createCheck3dConnectedEquipmentNoIndirectQuery` to get `query` at the end